java-grupa8/homework-1/tema/src/main/java8/ro/siit/tema/SalesOperations.java

```diff
@@ -0,0 +1,153 @@
+package ro.siit.tema;
+
+/**
+ * @author x-RauL-x
+ * A class that has all the methods who perform operations on the data
+ */
+
+public class SalesOperations {
```

Comment:
This code is part of another task(homework). It should not show up in this pull request.